src/server/routes/nyplApiClient/index.js

```diff
@@ -1,27 +1,35 @@
 import NyplApiClient from '@nypl/nypl-data-api-client';
-import config from '../../../../appConfig.js';
 import aws from 'aws-sdk';
 
+import config from '../../../../appConfig.js';
+import logger from '../../../../logger.js';
+
 const appEnvironment = process.env.APP_ENV || 'production';
+const kmsEnvironment = process.env.KMS_ENV || 'encrypted';
 const apiBase = config.api[appEnvironment];
-const kms = new aws.KMS({
-  region: 'us-east-1',
-});
+let decryptKMS;
+let kms;
 
-function decryptKMS(key) {
-  const params = {
-    CiphertextBlob: new Buffer(key, 'base64'),
-  };
+if (kmsEnvironment === 'encrypted') {
```

Comment:
Yeah, I can see how that can be confusing.
For the first `if`, I only wanted to declare the variables if it was in `encrypted` mode. Otherwise, it would try to call `aws.KMS(...)` and it could break.
I can update the second if statement if that helps you.